bananalyzer/data/fetch_schemas.py

```diff
@@ -1,331 +1,31 @@
-    firm: str = Field(description="Name of the law firm")
-    dates: str = Field(description="Duration of employment at the law firm")
-
-
-class AttorneyBarAdmission(BaseModel):
-    state: str = Field(description="State of bar admission")
-    year: str = Field(description="Year of bar admission")
-    country: str = Field(description="Country of bar admission", default="USA")
-
-
-class AttorneyEducation(BaseModel):
-    school: str = Field(description="Name of the educational institution")
-    year: str = Field(description="Year of graduation")
-    degree: str = Field(
-        description="Type of degree obtained",
-    )
-    honors: str = Field(
-        description="Honors received during education",
-    )
-
-
-class AttorneyAward(BaseModel):
-    date: str = Field(description="Date or duration when the award was received")
-    award: str = Field(description="Name of the award")
-    url: str = Field(
-        description="URL link to the award recognition",
-    )
-
-
-class AttorneySchema(BaseModel):
-    website: str = Field(description="URL of the attorney's profile page")
-    name: str = Field(description="Full name of the attorney")
-    title: str = Field(
-        description="Title of the attorney, such as Associate, Counsel, or Partner"
-    )
-    practice_areas_main: list[str] = Field(
-        description="Primary practice areas of the attorney"
-    )
-    practice_areas_all: list[str] = Field(description="Complete list of practice areas")
-    specialties: list[str] = Field(description="Specialized industry sectors")
-    email: str = Field(description="Email address of the attorney")
-    location: str = Field(description="Office location of the attorney")
-    phone: str = Field(
-        description="Direct phone number of the attorney",
-    )
-    bio: str = Field(description="Main bio description of the attorney")
-    experience: list[AttorneyExperience] = Field(
-        description="Past work history at other law firms"
-    )
-    matters: str = Field(description="List of past cases and work done by the attorney")
-    bar_admissions: list[AttorneyBarAdmission] = Field(
-        description="Bar admissions of the attorney"
-    )
-    law_school: list[AttorneyEducation] = Field(
-        description="Law school information of the attorney"
-    )
-    other_schools: list[AttorneyEducation] = Field(
-        description="Other education details of the attorney"
-    )
-    awards: list[AttorneyAward] = Field(
-        description="Awards and recognitions received by the attorney"
-    )
-    pdf_url: str = Field(
-        description="Link to a PDF bio of the attorney",
-    )
-    photo_url: str = Field(description="Link to the photo of the attorney")
-    news: list[str] = Field(
-        description="Links to news articles involving the attorney", default=[]
-    )
-
+    if fetch_id not in fetch_schemas:
+        raise ValueError(f"Invalid fetch_id: {fetch_id}")
```

Comment:
nit: add two basic parameterized test for both of these, one for the success case and one for the failure case